admin-app/src/data-provider.ts

```diff
@@ -0,0 +1,47 @@
+import { fetchUtils } from 'react-admin';
+
+const apiUrl = import.meta.env.VITE_HEX_ADMIN_API_URL;
+const httpClient = fetchUtils.fetchJson;
+
+export default {
+  getList: (resource: string) => {
+    const url = `${apiUrl}/${resource}/`;
+
+    return httpClient(url).then(({ json }) => ({
+      data: json,
+      total: json.length,
+    }));
+  },
+
+  getOne: (resource: any) => {
+    throw Error();
```

Comment:
```suggestion
    throw Error('This function is not yet implemented');
```

And I suggest making this replacement for all unimplemented methods of this file of course 😉